testsuite/tests/kuadrant/authorino/dinosaur/conftest.py

```diff
@@ -28,6 +29,18 @@ def admin_rhsso(blame, keycloak):
     return info
 
 
+@pytest.fixture(scope="module", autouse=True)
+def commit(request, authorization):
+    """Commits all important stuff before tests"""
```

Comment:
Mention in the comment this whole commit fixture is a temporary solution so it can be removed later. Should we track this with an github issue? If yes, maybe put a link in the comment as well.

Reply:
It does not need to be temporary, after the fix is done this will still work.